### DISCUSSION TITLE
Help scaling down the site

### DISCUSSION THREAD

**Discussion by TheNewbieWoodworker** [mmistakes/jekyll-theme-basically-basic#95] [Q&A] [answered]
I like the way my Basically Basic site looks when I scale it to 90% in Chrome, although I don't need to change the image sizes. So I thought I might be able to achieve something similar by changing some values in _variables.scss.

I changed $base-font-size and $max-font-size, and at first glance that was pretty close.

But some things didn't change. You'd probably know better than me, but here's what I noticed:

* The size of the heading font on a blog page doesn't change.
* The left margin (at least) of the top website icon and title doesn't change.

There's probably other things, but that's what I noticed. Can you tell me other settings I should change, or am I starting down a path that I should just avoid altogether?

Thanks.

**Comment by mmistakes** [comment, marked as answer]
Most of the base typography is set on the `html` element and bumps the `font-size` up or down depending on the screen width. Smaller point sizes for mobile, large for tablet, and even larger for desktop screens.

I'm using a [fluid type mixin](https://github.com/mmistakes/jekyll-theme-basically-basic/blob/master/_sass/basically-basic/functions/_fluid-type.scss) to do some "magic" here but you could just replace [this line](https://github.com/mmistakes/jekyll-theme-basically-basic/blob/master/_sass/basically-basic/_base.scss#L5-L7) with a non-mixin version if you want to shrink all of the type down:

```
html {
  font-size: 14px;
}
```

To reduce the widths of everything else.... well you're starting down the path of "just learn CSS" :stuck_out_tongue:

**Comment by TheNewbieWoodworker** [comment]
Thanks, and that's a perfect response. 😄I totally get it. I actually do know some CSS, but that's like saying I know Spanish because I can say "quesadilla".  Thanks again.
